### PR TITLE
New package: netease.identityv version 1.0.0.1

### PR DESCRIPTION
**Pull request by solomoncyj**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/50921)

### CODE REVIEW

**Comment by solomoncyj** [comment]
hello?
@denelon please manually start pipelines

**Comment by solomoncyj** [comment]
help

File: manifests/n/NetEase/identityv/1.0.0.1/netease.identityv.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created with YamlCreate.ps1 v2.1.0 $debug=NVS2.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: netease.identityv
```
Comment (ItzLevvie) [inline manifests/n/NetEase/identityv/1.0.0.1/netease.identityv.installer.yaml:4]:
```suggestion
PackageIdentifier: NetEase.IdentityV
```

File: manifests/n/NetEase/identityv/1.0.0.1/netease.identityv.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.1.0 $debug=NVS2.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: netease.identityv
```
Comment (ItzLevvie) [inline manifests/n/NetEase/identityv/1.0.0.1/netease.identityv.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: NetEase.IdentityV
```

File: manifests/n/NetEase/identityv/1.0.0.1/netease.identityv.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.1.0 $debug=NVS2.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: netease.identityv
```
Comment (ItzLevvie) [inline manifests/n/NetEase/identityv/1.0.0.1/netease.identityv.yaml:4]:
```suggestion
PackageIdentifier: NetEase.IdentityV
```

File: manifests/n/NetEase/identityv/1.0.0.1/netease.identityv.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+# PublisherSupportUrl: 
+# PrivacyUrl: 
+# Author: 
+PackageName: Identitiy V
+# PackageUrl: 
+License: ©Joker Studio ©2020 NetEaseInc. All Rights Reserved
+# LicenseUrl: 
+# Copyright: 
+# CopyrightUrl: 
+ShortDescription: Fear alwasys spring from the Hnknown
```
Comment (ItzLevvie) [inline manifests/n/NetEase/identityv/1.0.0.1/netease.identityv.locale.en-US.yaml:18]:
```suggestion
ShortDescription: Identity V is an asymmetrical multiplayer RPG video game developed and published by NetEase.
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Requires installation switches:
image

File: manifests/n/NetEase/identityv/1.0.0.1/netease.identityv.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+PackageIdentifier: netease.identityv
+PackageVersion: 1.0.0.1
+PackageLocale: en-US
+Publisher: Hong Kong NetEase Interactive Entertainment Limited
+# PublisherUrl: 
+# PublisherSupportUrl: 
+# PrivacyUrl: 
+# Author: 
+PackageName: Identitiy V
```
Comment (ItzLevvie) [inline manifests/n/NetEase/identityv/1.0.0.1/netease.identityv.locale.en-US.yaml:12]:
```suggestion
PackageName: IdentityV
```

File: manifests/n/NetEase/identityv/1.0.0.1/netease.identityv.locale.en-US.yaml [hunk 6]
```diff
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.1.0 $debug=NVS2.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: netease.identityv
+PackageVersion: 1.0.0.1
+PackageLocale: en-US
+Publisher: Hong Kong NetEase Interactive Entertainment Limited
```
Comment (ItzLevvie) [inline manifests/n/NetEase/identityv/1.0.0.1/netease.identityv.locale.en-US.yaml:7]:
```suggestion
Publisher: Netease, Inc.
```
This needs to match Control Panel.

File: manifests/n/NetEase/identityv/1.0.0.1/netease.identityv.locale.en-US.yaml (hunk 5 above)
Comment (ItzLevvie) [reply manifests/n/NetEase/identityv/1.0.0.1/netease.identityv.locale.en-US.yaml:12]: This needs to match Control Panel.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
[Manifest-Validation-Error](https://github.com/microsoft/winget-pkgs/labels/Manifest-Validation-Error): You need to update the folder and file names with correct capitalization.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
[Manifest-Validation-Error](https://github.com/microsoft/winget-pkgs/labels/Manifest-Validation-Error): You need to update the folder and file names with correct capitalization.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
[Manifest-Validation-Error](https://github.com/microsoft/winget-pkgs/labels/Manifest-Validation-Error): You need to update the folder and file names with correct capitalization.

`netease.identityV` (PackageIdentifier in file name) needs to be `NetEase.IdentityV` (PackageIdentifier in file name)
`identityV` (folder) needs to be `IdentityV` (folder)

**Comment by solomoncyj** [comment]
vs code is telling me it is an empty commit for some reason

**Comment by solomoncyj** [comment]
wtf?

**Pull request closed without merging**